implementations/rust/ockam/ockam_api/src/nodes/models/portal.rs

```diff
@@ -136,20 +136,25 @@ pub struct CreateOutlet<'a> {
     #[b(2)] pub worker_addr: Cow<'a, str>,
     /// A human-friendly alias for this portal endpoint
     #[b(3)] pub alias: Option<CowStr<'a>>,
+    /// Allow the outlet to be reachable from the default secure channel, useful when we want to
+    /// tighten the flow control
+    #[b(4)] pub reachable_from_default_secure_channel: bool,
```

Comment:
Should be `#[n(4)]`, but at this point I'm not sure if it's even handled somehow differently under the hood